compiler/ballerina-parser/src/main/java/io/ballerina/compiler/internal/parser/BallerinaParser.java

```diff
@@ -10721,15 +10726,16 @@ private STNode createFuncTypeQualNodeList(List<STNode> qualifierList, boolean ha
             } else if (qualifier.kind == SyntaxKind.ISOLATED_KEYWORD) {
                 validatedList.add(qualifier);
             } else if (qualifierList.size() == nextIndex) {
-                addInvalidNodeToNextToken(qualifier, DiagnosticErrorCode.ERROR_QUALIFIER_NOT_ALLOWED,
-                        ((STToken) qualifier).text());
+                functionKeyword = SyntaxErrors.cloneWithLeadingInvalidNodeMinutiae(functionKeyword, qualifier,
+                        DiagnosticErrorCode.ERROR_QUALIFIER_NOT_ALLOWED, ((STToken) qualifier).text());
             } else {
                 updateANodeInListWithLeadingInvalidNode(qualifierList, nextIndex, qualifier,
                         DiagnosticErrorCode.ERROR_QUALIFIER_NOT_ALLOWED, ((STToken) qualifier).text());
             }
         }
 
-        return STNodeFactory.createNodeList(validatedList);
+        STNode nodeList = STNodeFactory.createNodeList(validatedList);
```

Comment:
Any test case to cover this line?

Reply:
Seems the no coverage is false positive. It would be impossible to call `createFuncTypeQualNodeList()` without reaching this line right?

Reply:
Yeah :-D